### ISSUE TITLE
stable channel compatibility

### ISSUE THREAD

**Issue by johnpryan** [johnpryan/flutter_map#352]
Fix compatibility issues with [beta channel](https://github.com/flutter/flutter/commit/2e540931f73593e35627592ca4f9a4ca3035ed31)

**Comment by DavidKuennen** [comment]
It's the stable channel by now, so the name doesn't match anymore. ;-)

**Comment by johnpryan** [comment]
merging with https://github.com/johnpryan/flutter_map/issues/353

**Comment by MichalMisiaszek** [comment]
Any ETA on new version ? Unfortunately `flutter_image` on which map and plugin depends are failing too.
```file:///Users/michal/Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_image-1.0.0/lib/network.dart:75:31: Error: The argument type 'Null Function(StringBuffer)' can't be assigned to the parameter type 'Iterable<DiagnosticsNode> Function()'.
 - 'StringBuffer' is from 'dart:core'.
 - 'Iterable' is from 'dart:core'.
 - 'DiagnosticsNode' is from 'package:flutter/src/foundation/diagnostics.dart' ('file:///Users/michal/Development/flutter/packages/flutter/lib/src/foundation/diagnostics.dart').
Try changing the type of the parameter, or casting the argument to 'Iterable<DiagnosticsNode> Function()'.
        informationCollector: (StringBuffer information) {
                              ^
file:///Users/michal/Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_image-1.0.0/lib/network.dart:168:65: Error: The argument type 'String' can't be assigned to the parameter type 'DiagnosticsNode'.
 - 'DiagnosticsNode' is from 'package:flutter/src/foundation/diagnostics.dart' ('file:///Users/michal/Development/flutter/packages/flutter/lib/src/foundation/diagnostics.dart').
Try changing the type of the parameter, or casting the argument to 'DiagnosticsNode'.
      context: '$runtimeType failed to load ${instructions.uri}',

**Comment by MichalMisiaszek** [comment]
Perfect ! Thanks John, this is really great plugin.

**Comment by johnpryan** [comment]
😎 thanks for your patience, everyone

**Comment by johnpryan** [comment]
Released in version 0.5.6

**Comment by MichalMisiaszek** [comment]
Works ! :)
